popupwindow.js

```diff
@@ -0,0 +1,121 @@
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/multi1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Facebook",
+    image: "./imagsPop/facebook1.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+
+  {
+    heading: "Multi-post <br/>Stories",
+    image: "./imagsPop/project image 2.png",
+    content: `Lorem Ipsum is simply dummy text of the printing 
+		and typesetting industry. Lorem Ipsum has been the industry's 
+		standard dummy text ever since the 1500s, when an unknown 
+		printer took a galley of type and scrambled it to make a type 
+		specimen book. It has survived not only five centuries, but 
+		also the leap into electronic typesetting, remaining essent`,
+  },
+];
+
+const modal = document.querySelector(".modal");
+const popupWrap = document.querySelector(".pop-up-wrap");
+const projectBtn = document.querySelectorAll(".see-btn");
+
+projectBtn.forEach(function (Btn) {
+  Btn.addEventListener("click", function (e) {
+    const i = e.currentTarget.dataset.id;
+    popupWrap.innerHTML = `<div class="pop-up-nav">
+		  <h2 class="poppins" id="logo">${dataPopover[i].heading}</h2>
+		  <span class = "pop-up-closeBtn">&times;</span>
+		</div>
+	  
+	  <div class="pop-up-about">
+		  <ul class="poppins canopy-modal">
+			<li class="about-li" id="canopy">CANOPY</li>
+			<li class="about-li">Back End Dev</li>
+			<li class="about-li">2015</li>
+		  </ul>
+	  </div>
+	  
+	  <div class="pop-up-image-wrap">
+		<img class="snapshoot" src="${dataPopover[i].image}" alt="Tanner Christensen's portfolio image">
+	  </div>
+		 
+	  <div class="pop-up-text-content">
+		<p class="poppins-regular" id="modal-text">${dataPopover[i].content}</p>
+		<div class="tag-wrap">
+		<ul class="poppins-regular language-button">
+		  <li class="languages">html</li>
+		  <li class="languages">css</li>
+		  <li class="languages">javascript</li>
+		</ul>
+	  <div class="pop-up-btn">
+		<button class="poppins-regular see-live-btn">See live<img class="live-icon" src="./imagsPop/see-live-icon.svg"/></button>
+		<button class="poppins-regular see-source-btn">See source<img id="github-modal-icon" src="./images/Vector(2).svg"/></button>
+	  </div>
+	  </div>
+	  </div>
+	  `;
+    modal.style.visibility = "visible";
```

Comment:
- [ ]  it would be better if the overlay (background of the pop-up) were transparent as in the Figma design in order to match the Figma design 

- [ ] As you can see in the your picture it's responsive between mobile and desktop screen and right now your portfolio should be responsive on all screen so please try to fix that 

- YOurs 
<img width="555" alt="Screen Shot 2022-10-19 at 18 07 26" src="https://user-images.githubusercontent.com/61790587/196758576-dc1e3e57-bc26-4bc7-9e56-c99585eb31f0.png">
- Figma
<img width="240" alt="Screen Shot 2022-10-19 at 18 07 45" src="https://user-images.githubusercontent.com/61790587/196758618-fb6ca84d-1b65-49cb-b68b-80fdab35e0c3.png">